views/js/e2e/_helpers/commands/setupCommands.js

```diff
@@ -92,5 +115,19 @@ Cypress.Commands.add('setDeliveryForGuests', () => {
 
     // Wait until save happened properly
     // Not ideal but these requests have to be waited in this order upon delivery save
-    cy.wait(['@editDelivery', '@getData','@editDelivery', '@getData', '@editDelivery' ]);
-});
\ No newline at end of file
+    cy.wait(['@editDelivery', '@getData', '@editDelivery', '@getData', '@editDelivery', '@getData']);
+});
+
+Cypress.Commands.add('startTest', (testName) => {
+    cy.log('COMMAND: startTest', testName);
+
+    //windows workaround
+    cy.wait(1000);
```

Comment:
Needed on all platforms, as Cypress tries to click the link before eventListener is attached :)

Reply:
Ok.